#!/usr/bin/env python
PACKAGE = "racecar_control"

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, double_t

gen = ParameterGenerator()

gen.add("desire_velocity", double_t, 0, "Desire Velocity", 1.2, 0, 10.0)
gen.add("steering_gain", double_t, 0, "Steering Gain", 1.0, 0.1, 2.0)
gen.add("forward_distance", double_t, 0, "Pure Pursuit forward looking distance", 1.2, 0.1, 5.0)

exit(gen.generate(PACKAGE, "pure_pursuit", "pure_pursuit"))
